### PR TITLE
Rebase DJ-Tools

### PR DESCRIPTION
**Pull request by MiladNazeri**
- [x] Go to Rust-Experiments - Make sure you have correct permissions 

- [x] Run https://hifi-content.s3.amazonaws.com/milad/ROLC/Organize/O_Projects/Hifi/Scripts/hifi-content/DomainContent/Rust/DJ-Tools/DJ-Tools_Spawner.js

- [x] open up create and type in "Set_"

- [x] confirm you have 1 set of the following - 2 pad models, 2 pad boxes, 4 sound zones, 4 particles, 4 lights, 1 zone

- [x] Delete everything that says set and is unlocked 

- [x] Run the spawner again to confirm it creates another set of them

- [x] Enter the dj Zone and step up to the pads

- [x] Wave your left hand on the left pad

- [x] Did the left side effect come on?

- [x] Did you hear sound?

- [x] Moving from a high Y value to the model, do particle's radius get thicker? (Up and down)

- [x] Moving from close to you to away from you on the z, does the color change?

- [x] Do the same for the Right side and see if the effect comes on the same but reversed on x

- [x] Does it turn on when you wave your hands in and turn off when you wave them off

- [x] BookMark this location

- [x] Go to another domain and bookmark that location

- [x] go back to the previous Bookmark

- [x] Wave your hand in front of the pad and while keeping one hand there, go to the other bookmark

- [x] Wait 3 seconds

- [x] Go back to the previous bookmark

- [x] Did the effect stop running?

- [x] Go back to the pads.  Wave two hands and see if they are still working.

- [x] Remove one hand and make sure it doesn't turn off

- [x] Get a second person

- [x] Both people try and affect different pads and see if they work

- [x] Both people try and affect one pad and see if it still works

### CODE REVIEW

**Comment by AndySeattle** [comment]
@MiladNazeri we both reviewed / tested these, are you going to be releasing a different version? / Do you need me to comment further on these? do you want to do a second pass on these before we submit them?

**Comment by MiladNazeri** [comment]
Hi @AndySeattle, I just updated the code to match a previous code review comments, as well as use a centralized helper script.  I'm not worried about the multiple people part, but can you run through everyone one more time when you have a moment?

**Comment by AndySeattle** [comment]
We saw the server scripts stop once, but once we figured that out, and also disabled debug, we were good to go! stamp and passed.

File: DomainContent/Rust/DJ-Tools/DJ-Tools_Personal_WIP.js [hunk 1]
```diff
@@ -416,9 +417,9 @@
             userData.grabbableKey = { grabbable: false };
             userData.performance.DEBUG = DEBUG;
             stringified = JSON.stringify(userData);
-            entID = createParticleEntity(name, particlePosition, stringified, dispatchZoneID);
+            entityID = createParticleEntity(name, particlePosition, stringified, dispatchZoneID);
             entID2 = createParticleEntity(name2, particlePosition2, stringified, dispatchZoneID);
```
Comment (misslivirose) [inline DomainContent/Rust/DJ-Tools/DJ-Tools_Personal_WIP.js]: Avoid abbreviating variable names

Comment (misslivirose) [reply DomainContent/Rust/DJ-Tools/DJ-Tools_Personal_WIP.js]: (for both entity variables and the list of 'allEnts'

File: DomainContent/Rust/DJ-Tools/DJ_Sensor_Box_Client.js [hunk 2]
```diff
@@ -116,6 +117,11 @@
                 DEBUG = userdataProperties.performance.DEBUG;
                 if (DEBUG) {
                     debugCubeID = Entities.findEntitiesByName("Set_Phlash_Debug-Cube", position, 10)[0];
```
Comment (misslivirose) [inline DomainContent/Rust/DJ-Tools/DJ_Sensor_Box_Client.js]: 10 - magic number, define as debug search radius

**Comment by MiladNazeri** [comment]
This one is fixed now and the personal version is no longer a wip

**Pull request merged**